### ISSUE TITLE
[BITV] 9.1.4.11/7.2 - When selecting a new color for the background, which is indicated by a checkmark, this checkmark does not meet the contrast requirements, as the contrast ratio with the background is below 3:1. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41861]
![image](https://github.com/nextcloud/server/assets/6078378/13643dd4-5a8e-4843-ab08-81aeb3302cf1)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v7-n2

**Comment by susnux** [comment]
vue component issue, probably the check icon colors needs to be calculated from the background color

**Comment by ShGKme** [comment]
And probably we should add the same border as on background selector

**Comment by emoral435** [comment]
Once #42499 gets merged, this should be closed :)
